ext/search_kit/Civi/Api4/Action/SearchDisplay/Run.php

```diff
@@ -108,7 +108,12 @@ protected function processResult(SearchDisplayRunResult $result) {
         }
     }
 
-    $apiResult = civicrm_api4($entityName, 'get', $apiParams, $index);
+    try {
+      $apiResult = civicrm_api4($entityName, 'get', $apiParams, $index);
+    } catch (\Throwable $e) {
+      \Civi::log()->error("SearchDisplay.Run error: " . get_class($e) . ": {$entityName}.get: [id]" . $this->display['id'] . ' [saved_search_id]' . $this->display['saved_search_id'] . ' [label] ' . $this->display['label']);
```

Comment:
Maybe specify display id is the id you mean?

```suggestion
      \Civi::log()->error("SearchDisplay.Run error: " . get_class($e) . ": {$entityName}.get: [display id] " . $this->display['id'] . ' [saved_search id] ' . $this->display['saved_search_id'] . ' [label] ' . $this->display['label']);
```